newdle/client/src/components/ParticipantTable.js

```diff
@@ -70,7 +70,7 @@ function AvailabilityRow({availability: {startDt, available}, setActiveDate, act
       onClick={() => setActiveDate(startDt)}
       active={active}
     >
-      <Table.Cell width={3}>
+      <Table.Cell width={200} className={styles['schedule-info']}>
```

Comment:
the value you passed as the `width` prop is invalid (https://react.semantic-ui.com/collections/table/)